types/react-fa/index.d.ts

```diff
@@ -37,7 +37,6 @@ export const Icon: ComponentClass<IconProps>;
 
 export interface IconStackProps extends SizeOverrideHTMLProps<IconStack> {
     size?: IconSize;
-    children?: IconProps[];
```

Comment:
a bit of an eye roll that after all this discussion, it was just removed anyways in a42c34d.

I'll PR the real fix shortly.